Source/MissionSystem/Private/MSMissionSystemComponent.cpp

```diff
@@ -349,6 +325,28 @@ void UMSMissionSystemComponent::ClearMissionHistory()
     MissionHistory.Clear();
 }
 
+void UMSMissionSystemComponent::TryResumeMissionFromHistory()
+{
+#if !( UE_BUILD_SHIPPING || UE_BUILD_TEST )
+    if ( CVarDisableAllMissions.GetValueOnGameThread() == 1 )
+    {
+        return;
+    }
+#endif
+
+    if ( bTryResumeMissionFromHistory )
```

Comment:
Shouldn't this if statement be in BeginPlay and then call this function?

Reply:
no because it may happen that in BeginPlay, the mission history has not been loaded yet, and so the initial mission would always be started